### ISSUE TITLE
Impossible to use with Django

### ISSUE THREAD

**Issue by jerry-pca** [dagster-io/dagster#13636]
### What's the issue or suggestion?

Hello I believe there is no way to use dagster with django unless I am wrong. I looked at the documentation and there is nothing that indicates this should be possible. As of right now users who are transitioning from celery jobs to dagster appear to be blocked.

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by will-holley** [comment]
Can you describe how you intend to use Django with Dagster?

**Comment by timothedelion** [comment]
I guess @jerry-pca means a way to use Django ORM from within Dagster tasks.

There are some answers in this SO : https://stackoverflow.com/questions/59048161/integrating-dagster-with-django
TLDR : Run `django.setup()` and provide the python file the path toward the Django project.

@ Dagster team : Is it the way you would recommend ?

**Comment by sryza** [comment]
Dagster tasks should be able to execute any Python code
